src/lib.rs

```diff
@@ -60,19 +60,14 @@ impl LightningD {
         let rpcconnect = format!("--bitcoin-rpcconnect={}", bitcoind.params.rpc_socket.ip());
         let rpcport = format!("--bitcoin-rpcport={}", bitcoind.params.rpc_socket.port());
 
-        let (user, pass) = bitcoind
+        let cookie = bitcoind
             .params
             .get_cookie_values()
-            .expect("failing get_cookie_values");
-
-        let rpcuser = format!(
-            "--bitcoin-rpcuser={}",
-            user.expect("missing user in cookie file")
-        );
-        let rpcpassword = format!(
-            "--bitcoin-rpcpassword={}",
-            pass.expect("missing password in cookie file")
-        );
+            .expect("failing get_cookie_values")
+            .expect("missing cookie file");
```

Comment:
the first expect should be a `?` The second should be mapped to a specific error